english/marshmallow.md

```diff
@@ -1,5 +1,9 @@
 Once upon a time many authors collaborated to make meaningless linked story.
 
+So each one of these authors used what they learend in the course to fork [this](https://github.com/udacity/create-your-own-adventure) repository 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.